pkg/controller/metrics/metrics.go

```diff
@@ -105,6 +106,31 @@ var (
 			Buckets: []float64{60, 90, 180, 300, 600, 1200},
 		},
 		[]string{"cluster_version", "platform", "cluster_pool_namespace", "cluster_pool_name"})
+	// MetricStoppingClusters is a prometheus metric that tracks the transition time for currently stopping clusters
+	MetricStoppingClusters = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "hive_cluster_deployments_stopping",
```

Comment:
I believe metric names are supposed to have units.

...[yeah](https://prometheus.io/docs/practices/naming/)
```suggestion
			Name:    "hive_cluster_deployments_stopping_seconds",
```